src/custom/state/claim/reducer.ts

```diff
@@ -64,7 +64,7 @@ export type ClaimState = {
   // claiming
   claimStatus: ClaimStatus
   claimedAmount: string
-  estimatedGas: string
+  estimatedGas?: string
```

Comment:
@anxolin Had to set it as optional, because I was getting a Typescript error. Please review.